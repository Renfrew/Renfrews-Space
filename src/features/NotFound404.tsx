import React, { ReactElement } from 'react';
import Container from '@material-ui/core/Container';

const NotFound404 = (): ReactElement => {
  return (
    <Container>
      <p>Not Found</p>
    </Container>
  );
};

export default NotFound404;
