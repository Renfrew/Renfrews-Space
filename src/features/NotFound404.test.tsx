import React from 'react';
import renderer from 'react-test-renderer';
import NotFound404 from './NotFound404';

describe('Testing the NotFound Page', () => {
  it('The NotFound page renders correctly', () => {
    const tree = renderer.create(<NotFound404 />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
