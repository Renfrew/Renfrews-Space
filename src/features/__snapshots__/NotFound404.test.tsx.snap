// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the NotFound Page The NotFound page renders correctly 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg css-cnms5y-MuiContainer-root"
>
  <p>
    Not Found
  </p>
</div>
`;
